src/ifcblenderexport/blenderbim/bim/ui.py

```diff
@@ -804,6 +801,51 @@ def draw(self, context):
         row.prop(props, "swept_solids")
 
 
+class BIM_PT_presentation_layer_data(Panel):
+    bl_label = "IFC Presentation Layers"
+    bl_idname = "BIM_PT_presentation"
+    bl_space_type = "PROPERTIES"
+    bl_region_type = "WINDOW"
+    bl_context = "data"
+
+    @classmethod
+    def poll(cls, context):
+        return (
+            context.active_object is not None
+            and context.active_object.type == "MESH"
+            and hasattr(context.active_object.data, "BIMMeshProperties")
+        )
+
+    def draw(self, context):
+        if not context.active_object.data:
+            return
+        layout = self.layout
+        elem_props = context.active_object.BIMObjectProperties.presentation_layer
+        scene_props = context.scene.BIMProperties
+
+        if elem_props.name != "":
+            layout.label(text=f'Object is part of Presentation layer "{elem_props.name}"')
+            layout.row().operator("bim.remove_from_presentation_layer")
+        else:
+            layout.label(text="Not included in any presentation layer")
+
+        if scene_props.presentation_layers:
+            layout.template_list(
+                "BIM_UL_presentation_layers",
```

Comment:
Switched to BIM_UL_generic. In the future, when we create a more special UL which has the typical UI like an "eye" icon for visibility and snow icon for freeze we can switch this back.